File: web-example/src/api/evt/CreateOrderEvt.ts
    import {BaseEvt} from "@/src/api/evt/BaseEvt";

/**
 * 创建订单
**/

export interface  CreateOrderEvt extends BaseEvt {

        /**
            *订单ns
            *输入字符串的最小长度为：0，输入字符串的最大长度为：50
            *在java中的类型为：String
        **/
        sn?: string;
        /**
            *订单总价
            *必填项，不能为空
            *在java中的类型为：Integer
        **/
        totalAmount: number;
}

File: examples/swagger-3/src/main/java/com/wuxp/codegen/swagger3/example/maven/evt/CreateOrderEvt.java
package com.wuxp.codegen.swagger3.example.maven.evt;


import com.wuxp.codegen.swagger3.example.maven.enums.Sex;
import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;
import java.beans.Transient;
import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Map;

@Data
public class CreateOrderEvt extends BaseEvt {

    private static DateFormat dateFormat = new SimpleDateFormat();

    @Size(max = 50)
    private String sn;

    @NotNull
    private transient Integer totalAmount;

    private Map<Sex, String> test;

    @Transient
    public Integer getTotalAmount() {
        return totalAmount;
    }
}
